__tests__/app_provider.ts

```diff
@@ -60,7 +62,7 @@ type KnexQueryResult = Array<{[attributeName: string]: any}>;
 const resolvers = {
     ...connectionResolvers,
     Query: {
-        async users(_: any, {input: inputArgs}: {input: IInputArgs}) {
+        async users(_: any, input: IInputArgs) {
```

Comment:
This was an unnoticed issue in the tests where input was not being extracted by the resolver correctly.